pyproject.toml

```diff
@@ -32,6 +32,10 @@ zstandard = ">=0.17,<0.19"
 python-can = "^4.0.0"
 tabulate = "^0.8.9"
 construct = "^2.10.68"
+numpy = "^1.21.4"
+openpyxl = "^3.0.9"
+pandas = "^1.3.4"
+matplotlib = "^3.4.3"
```

Comment:
I would suggest that we add this as extra dependencies and make the analyzer optional. Otherwise these heavy dependencies are always present.

Reply:
Good idea.
For the cli interface we need to know, if we can add the Analyzer to the cli or not.
Is there a convenient way to detect if the packet was installed with `extra`?
Or do we need something like
```py
try:
    from gallia.analyzer import Analyzer
    load_module(Analyzer)
except ModuleNotFoundError:
    print("Install extra dependencies to use all gallia features")
```

Reply:
done